### PR TITLE
Problem: Events are insufficient for monitoring gravity transfer

### PR DESCRIPTION
**Pull request by calvinaco**
Solution: Emit events in AfterSendToCosmosEvent hook

This PR is related to https://github.com/crypto-org-chain/gravity-bridge/pull/12

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by yihuang — APPROVED** [review]

**Comment by yihuang** [comment]
there's a lint issue @calvinaco

File: x/cronos/types/events.go [hunk 1]
```diff
@@ -9,12 +9,16 @@ import (
...
+	AttributeKeyAmount                = "amount"
+	AttributeKeyReceiver              = "receiver"
+	AttributeKeyEthereumTokenContract = "ethereum_token_contract"
 
 	// events
-	EventTypeConvertVouchers = "convert_vouchers"
-	EventTypeTransferTokens  = "transfer_tokens"
+	EventTypeConvertVouchers             = "convert_vouchers"
+	EventTypeTransferTokens              = "transfer_tokens"
+	EventTypeEthereumSendToCosmosHandled = "ethereum_send_to_cosmos_handled"
```
Comment (thomas-nguy) [inline x/cronos/types/events.go:21]: `ethereum_to_cronos_handled` to be more accurate?

or `ethereum_to_crc20_handled`

Comment (calvinaco) [reply x/cronos/types/events.go:21]: Depends on whether we want a more generic event type or specific one. If the long run we want to push https://github.com/crypto-org-chain/gravity-bridge/pull/12 to upstream. Then it is better to stick with the Gravity contract event name.

Comment (thomas-nguy) [reply x/cronos/types/events.go:21]: yeah but the handled do also gravity->crc20 conversion which is not exactly only sendToCosmos event

Comment (thomas-nguy) [reply x/cronos/types/events.go:21]: or move the event firing at the beginning of the function

Comment (calvinaco) [reply x/cronos/types/events.go:21]: updated.

**Review by thomas-nguy — APPROVED** [review]

**Pull request merged**